global/cookie-banner.js

```diff
@@ -71,5 +73,10 @@
     }
   )
 
-  setTimeout(loadCookieBanner, 3000);
+  /* TODO: @team:growth @kay @due:2024-06-01
```

Comment:
We'll need to work on the syntax a bit before we add an automated check for this type of thing, but for now it is helpful 😁

Reply:
Definitely 🤣 for now it's easy enough to search and replace the things - still working on the docs 👀